packages/web/src/components/Container/_tools.scss

```diff
@@ -0,0 +1,12 @@
+@use '../../tools/breakpoint';
+@use 'sass:map';
```

Comment:
Built-in modules should be listed first (looks like Stylelint is sleeping here…).